openedx/features/learner_analytics/views.py

```diff
@@ -96,19 +96,34 @@ def get(self, request, course_id):
             schedule_data = self.get_assignments_with_due_date(request, course_key)
             (grade_data, schedule_data) = self.sort_grade_and_schedule_data(grade_data, schedule_data)
 
+            # The following try blocks are hacks to provide a more robust response if the Analytics API errors
```

Comment:
Please use the TODO style and replace with.
```
# TODO: LEARNER-3854: Fix hacked defaults with real error handling if implementing Learner Analytics.
```
Nit: I'd move it immediately next to try block.